test/claim-iat.test.js

```diff
@@ -53,22 +39,32 @@ describe('issue at', function() {
       {foo: 'bar'},
     ].forEach((iat) => {
       it(`should error with iat of ${util.inspect(iat)}`, function (done) {
-        expect(() => signWithIssueAtSync(iat, {})).to.throw('"iat" should be a number of seconds');
-        signWithIssueAtAsync(iat, {}, (err) => {
-          expect(err.message).to.equal('"iat" should be a number of seconds');
-          done();
+        signWithIssueAt(iat, {}, (err) => {
+          // the sign function swallows the error that is thrown by expect, so use done(err)
+          try {
+            expect(err).to.be.instanceOf(Error);
+            expect(err.message).to.equal('"iat" should be a number of seconds');
+            done();
+          }
+          catch(e) {
+            done(e);
+          }
         });
       });
     });
 
     // undefined needs special treatment because {} is not the same as {iat: undefined}
     it('should error with iat of undefined', function (done) {
-      expect(() => jwt.sign({iat: undefined}, undefined, {algorithm: 'none'})).to.throw(
-        '"iat" should be a number of seconds'
-      );
-      jwt.sign({iat: undefined}, undefined, {algorithm: 'none'}, (err) => {
-        expect(err.message).to.equal('"iat" should be a number of seconds');
-        done();
+      testUtils.signJWTHelper({iat: undefined}, 'secret', {algorithm: 'none'}, (err) => {
+        // the sign function swallows the error that is thrown by expect, so use done(err)
+        try {
+          expect(err).to.be.instanceOf(Error);
+          expect(err.message).to.equal('"iat" should be a number of seconds');
+          done();
+        }
+        catch(e) {
+          done(e);
+        }
```

Comment:
maybe we can move the try-catch to the helper function, otherwise all tests will need to add it.
The bad part is that if the test has a a line like the 61, it won't be clear what failed. 🤔

Reply:
I will see what I can do to break this out into a helper. Perhaps I can make it part of the `signJWTHelper` function.

Reply:
Addressed in https://github.com/auth0/node-jsonwebtoken/pull/523/commits/731a76db5b78c21c73ff53a514bdb9c8081432b4 before commit was squashed.

Reply:
I also applied the helper function to the verify tests even thought they do not use `once`. This was to keep the tests consistent and to also address this issue reappearing if the callback for verify is every made asynchronous.